flashinfer/experimental/sm12x/_lib/env.py

```
@@ -0,0 +1,79 @@
+    sync_legacy_env()
+    return os.environ.get(PREFIX + suffix)
+
+
+def env_flag(suffix: str, *, default: bool = False) -> bool:
+    raw = env_raw(suffix)
+    if raw is None:
+        return default
+    return raw.strip().lower() not in {"", "0", "false", "no", "off"}
+
+
+__all__ = ["PREFIX", "sync_legacy_env", "env_raw", "env_flag"]
```

Ruff (0.15.21)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)